src/boot/efi/fdt.c

```diff
@@ -0,0 +1,1289 @@
+// SPDX-License-Identifier: (GPL-2.0-or-later OR BSD-2-Clause)
+/*
+ * Adapted from libfdt - Flat Device Tree manipulation
+ * Copyright (C) 2006 David Gibson, IBM Corporation.
+ * Copyright 2021 Emil Renner Berthing
+ */
+
+#include <stdbool.h>
```

Comment:
hmm, are you sure this is safe to include? isn't this a libc-specific header? which we don#t have in efi?

Reply:
Yes, as far as i know, this is provided by the compiler.